### PR TITLE
Show README content as package description on PyPI

### PR DESCRIPTION
**Pull request by kynan**
Also change image URLs to point to GitHub so they work on PyPI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,11 +23,11 @@
 
 Diffing notebooks in the terminal:
 
-![terminal-diff](docs/source/images/nbdiff-terminal.png)
+![terminal-diff](https://github.com/jupyter/nbdime/raw/master/docs/source/images/nbdiff-terminal.png)
```
Comment (vidartf) [inline README.md]: Won't this break the images in older releases on PyPI if these are moved in master? Ideally the code that puts this inti `setup.py` should get the current git hash and use that to create a permalink?

Comment (kynan) [reply README.md]: Good point, changed this to current Git hash.

**Comment by vidartf** [comment]
Would it make more sense to base it on version tags? E.g. 
```python
readme_content = readme_content.replace(
    'docs/source/images', 
    'https://github.com/jupyter/nbdime/raw/{version}/docs/source/images/nbdiff-web.png'.format(version=version)
)
```

**Comment by vidartf** [comment]
Also note that the CI failure on 2.7 (even after rebuild) does not seem to happen on master. Unsure why. Restarted it again, will see.

**Comment by kynan** [comment]
> Would it make more sense to base it on version tags? E.g.
> 
> ```python
> readme_content = readme_content.replace(
>     'docs/source/images', 
>     'https://github.com/jupyter/nbdime/raw/{version}/docs/source/images'.format(version=version)
> )
> ```

Great suggestion, done!

**Comment by kynan** [comment]
The 2.7 test seems to be failing due to non-pinned dependencies no longer supporting Python 2:
```
Compiling /tmp/pip-install-5YOY8h/jinja2/jinja2/asyncfilters.py ...
    File "/tmp/pip-install-5YOY8h/jinja2/jinja2/asyncfilters.py", line 7
      async def auto_to_seq(value):
              ^
  SyntaxError: invalid syntax
  
  Compiling /tmp/pip-install-5YOY8h/jinja2/jinja2/asyncsupport.py ...
    File "/tmp/pip-install-5YOY8h/jinja2/jinja2/asyncsupport.py", line 22
      async def concat_async(async_gen):
              ^
  SyntaxError: invalid syntax

  changing mode of /home/travis/virtualenv/python2.7.15/bin/jupyter-nbconvert to 775
  changing mode of /home/travis/virtualenv/python2.7.15/bin/jupyter-run to 775
  changing mode of /home/travis/virtualenv/python2.7.15/bin/jupyter-kernelspec to 775
  changing mode of /home/travis/virtualenv/python2.7.15/bin/jupyter-kernel to 775

  Compiling /tmp/pip-install-5YOY8h/pexpect/pexpect/_async.py ...
    File "/tmp/pip-install-5YOY8h/pexpect/pexpect/_async.py", line 20
      transport, pw = yield from asyncio.get_event_loop()\
                               ^
  SyntaxError: invalid syntax

  changing mode of /home/travis/virtualenv/python2.7.15/bin/ipython to 775
  changing mode of /home/travis/virtualenv/python2.7.15/bin/iptest2 to 775
  changing mode of /home/travis/virtualenv/python2.7.15/bin/iptest to 775
  changing mode of /home/travis/virtualenv/python2.7.15/bin/ipython2 to 775
  Compiling /tmp/pip-install-5YOY8h/ipykernel/ipykernel/tests/_asyncio.py ...
    File "/tmp/pip-install-5YOY8h/ipykernel/ipykernel/tests/_asyncio.py", line 14
      async def task():
              ^
  SyntaxError: invalid syntax

  Compiling /tmp/pip-install-5YOY8h/notebook/notebook/tests/selenium/utils.py ...
    File "/tmp/pip-install-5YOY8h/notebook/notebook/tests/selenium/utils.py", line 206
      def append(self, *values, cell_type="code"):
                                        ^
  SyntaxError: invalid syntax
```

**Comment by vidartf** [comment]
> The 2.7 test seems to be failing due to non-pinned dependencies no longer supporting Python 2:

I just cleared the travis build cache, and re-ran a build on master, [which succeeded](https://travis-ci.org/jupyter/nbdime/builds/613439900). One of the errors are also from jinja2, but I see that the same version is being installed in both runs?

**Pull request merged**